test/e2e/spec/test_configuration_spec.js

```diff
@@ -0,0 +1,18 @@
+var chakram = require('./../setup.js').chakram;
+var expect = chakram.expect;
+
+describe("test configuration", function () {
+
+    it("has a hostname", function () {
+        return expect(process.env.BOILERPLATE_DOMAIN).to.not.equal('');
```

Comment:
Do we have `BOILERPLATE_` variables here? (I thought they're called `DESECSTACK_`)

Reply:
fixed (also fixed the reason why those tests didn't fail)